CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## [1.44.101](https://github.com/brave/brave-browser/releases/tag/v1.44.101)
+
+ - Added .zil TLD support for Unstoppable Domains. ([#24621](https://github.com/brave/brave-browser/issues/24621))
+ - Added "Prevent fingerprinting via language settings" under the "Brave Shields & privacy" settings. ([#24621](https://github.com/brave/brave-browser/issues/23919))  
+ - Added Solana account creation dialog in Brave Wallet under the "Send" tab if Solana account doesn't exist. ([#24463](https://github.com/brave/brave-browser/issues/24463))  
+ - Updated Brave Wallet to create a Solana account by default when creating or restoring a wallet. ([#25432](https://github.com/brave/brave-browser/issues/25432)) 
+ - Improved de-AMP functionality. ([#22917](https://github.com/brave/brave-browser/issues/22917))
+ - Removed support for legacy Brave Rewards anonymous Uphold cards. ([#24464](https://github.com/brave/brave-browser/issues/24464))
+ - Removed 2 BAT minimum threshold under Brave Rewards before being able to use Uphold as a custodial provider. ([#24759](https://github.com/brave/brave-browser/issues/24759))
+ - Fixed pending Solana Devnet requests not being displayed under "Transactions" via Brave Wallet. ([#24410](https://github.com/brave/brave-browser/issues/24410))
+ - Fixed extra whitespace being added when pasting a contract address while adding a custom token in Brave Wallet. ([#24728](https://github.com/brave/brave-browser/issues/24728))
```

Comment:
Removed extra whitespace instead of Fixed extra whitespace

Reply:
Done 👍